### PR TITLE
nebula netflixoss 11.3.1

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,9 +1,17 @@
+buildscript {
+    dependencies {
+        classpath ('com.fasterxml.jackson:jackson-bom:') version {
+            strictly('2.9.10.+')
```
Comment (brharrington) [inline build.gradle:4]: Why is this needed?

Comment (sullis) [reply build.gradle:4]: Without the build script dependency:

```
> Could not resolve all files for configuration ':classpath'.
   > Could not resolve com.fasterxml.jackson:jackson-bom:{strictly 2.9.10.20210106}.
     Required by:
         project : > com.netflix.nebula.netflixoss:com.netflix.nebula.netflixoss.gradle.plugin:11.3.0 > com.netflix.nebula:gradle-netflixoss-project-plugin:11.3.0 > com.netflix.nebula:gradle-dependency-lock-plugin:13.2.0
      > Cannot find a version of 'com.fasterxml.jackson:jackson-bom' that satisfies the version constraints:
           Dependency path ':spectator:unspecified' --> 'com.netflix.nebula.netflixoss:com.netflix.nebula.netflixoss.gradle.plugin:11.3.0' (runtime) --> 'com.netflix.nebula:gradle-netflixoss-project-plugin:11.3.0' (runtimeElements) --> 'com.netflix.nebula:gradle-dependency-lock-plugin:13.2.0' (runtimeElements) --> 'com.fasterxml.jackson:jackson-bom:{strictly 2.9.10.20210106}'
           Dependency path ':spectator:unspecified' --> 'com.netflix.nebula.netflixoss:com.netflix.nebula.netflixoss.gradle.plugin:11.3.0' (runtime) --> 'com.netflix.nebula:gradle-netflixoss-project-plugin:11.3.0' (runtimeElements) --> 'com.netflix.nebula:nebula-release-plugin:17.2.0' (runtimeElements) --> 'com.fasterxml.jackson:jackson-bom:2.11.0'

   > Could not resolve com.fasterxml.jackson:jackson-bom:2.11.0.
     Required by:
         project : > com.netflix.nebula.netflixoss:com.netflix.nebula.netflixoss.gradle.plugin:11.3.0 > com.netflix.nebula:gradle-netflixoss-project-plugin:11.3.0 > com.netflix.nebula:nebula-release-plugin:17.2.0
      > Cannot find a version of 'com.fasterxml.jackson:jackson-bom' that satisfies the version constraints:
           Dependency path ':spectator:unspecified' --> 'com.netflix.nebula.netflixoss:com.netflix.nebula.netflixoss.gradle.plugin:11.3.0' (runtime) --> 'com.netflix.nebula:gradle-netflixoss-project-plugin:11.3.0' (runtimeElements) --> 'com.netflix.nebula:gradle-dependency-lock-plugin:13.2.0' (runtimeElements) --> 'com.fasterxml.jackson:jackson-bom:{strictly 2.9.10.20210106}'
           Dependency path ':spectator:unspecified' --> 'com.netflix.nebula.netflixoss:com.netflix.nebula.netflixoss.gradle.plugin:11.3.0' (runtime) --> 'com.netflix.nebula:gradle-netflixoss-project-plugin:11.3.0' (runtimeElements) --> 'com.netflix.nebula:nebula-release-plugin:17.2.0' (runtimeElements) --> 'com.fasterxml.jackson:jackson-bom:2.11.0'

```

Is there a better way to resolve this?

Comment (brharrington) [reply build.gradle:4]: @rpalcolea do you have a suggestion here? It seems like the nebula components have inconsistent BOM requirements.

Comment (rpalcolea) [reply build.gradle:4]: oh I totally forgot about this last week @brharrington , sorry about that

I just released 11.3.1 of the netflixoss plugin and resolution strategy won't be necessary. We were leaking Gradle's enforced platform for jackson bom and that caused the conflict in the buildscript classpath but should be good now

Comment (sullis) [reply build.gradle]: Awesome. I just upgraded this PR to version 11.3.1

**Review by brharrington — APPROVED** [review]

**Comment by brharrington** [comment]
Thanks @sullis !
